src/com/codebusters/game/Viewer.java

```diff
@@ -79,45 +88,73 @@ public Viewer() {
 
         //text panel for main story
```

Comment:
these simple comments the way you break up the code is nice.  Makes finding code quick.